### PR TITLE
docs(core-concepts): getters.md and actions.md translated

### PR DESCRIPTION
**Pull request by raulwwq0**
## Pull request type

Please check the type of change your PR introduces:

- [ ] Feature :sparkles:
- [ ] Code style update (formatting, renaming) :art:
- [ ] Refactoring (no functional changes, no api changes) :recycle:
- [X] Documentation content changes :memo:
- [ ] Other :zap:

## What is the new behavior?

Added translation to `getters.md` and `actions.md`.

## Does this introduce a breaking change?

- [ ] Yes
- [X] No

### CODE REVIEW

File: core-concepts/actions.md [hunk 1]
```diff
@@ -1,22 +1,19 @@
...
-# Actions
+# Acciones {#actions}
 
 <VueSchoolLink
   href="https://vueschool.io/lessons/synchronous-and-asynchronous-actions-in-pinia"
   title="Learn all about actions in Pinia"
 />
 
-Actions are the equivalent of [methods](https://v3.vuejs.org/guide/data-methods.html#methods) in components. They can be defined with the `actions` property in `defineStore()` and **they are perfect to define business logic**:
+Las acciones son el equivalente de los [métodos](https://v3.vuejs.org/guide/data-methods.html#methods) en los componentes. Pueden ser definidos con la propiedad `actions`en `defineStore()` y **son perfectas para definir lógica de negocio**:
```
Comment (edgonzalez24) [inline core-concepts/actions.md:8]: Add a space between `actions` y 'en'. 
E.g. (`actions` en)

File: core-concepts/actions.md [hunk 2]
```diff
@@ -110,17 +107,17 @@ export default {
...
-## Usage with the Options API
+## Uso con la API de Opciones {#usage-with-the-options-api}
 
 <VueSchoolLink
   href="https://vueschool.io/lessons/access-pinia-actions-in-the-options-api"
   title="Access Pinia Getters via the Options API"
 />
 
-For the following examples, you can assume the following store was created:
+Para los próximos ejemplos supón que el siguiente almacén fue creado:
```
Comment (edgonzalez24) [inline core-concepts/actions.md:117]: I think that we can improve to: "Para los siguientes ejemplos, puedes asumir que se ha creado el siguiente almacén:". Do you agree?

File: core-concepts/getters.md [hunk 3]
```diff
@@ -115,11 +113,11 @@ export default {
 </script>
 
 <template>
-  <p>User 2: {{ getUserById(2) }}</p>
+  <p>Usuario 2: {{ getUserById(2) }}</p>
```
Comment (edgonzalez24) [inline core-concepts/getters.md:116]: I think it is not necessary to add the translation to the code examples. For didactic reasons.

File: core-concepts/getters.md [hunk 4]
```diff
@@ -215,23 +213,23 @@ export default {
 }
 ```
 
-### Without `setup()`
+### Sin `setup()` {#without-setup}
 
-You can use the same `mapState()` function used in the [previous section of state](./state.md#options-api) to map to getters:
+Puedes usar la misma función `mapState()` usada en la [sección anterior del estado](./state.md##usage-with-the-options-api) para mapear los getters:
```
Comment (edgonzalez24) [inline core-concepts/getters.md:218]: Check the correct URL

File: core-concepts/getters.md (hunk 3 above)
Comment (raulwwq0) [reply core-concepts/getters.md:116]: Alright, I will change other translation in code examples then

**Comment by raulwwq0** [comment]
@edgonzalez24 can you review the fixes please?

**Review by edgonzalez24 — APPROVED** [review]
LGTM! 💯

**Pull request merged**